kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/vault/Vault.java

```diff
@@ -88,6 +89,27 @@ public boolean isAvailable() {
         }
     }
 
+    /**
+     * Gets the installed version.
+     *
+     * @return the version
+     */
+    public String getVersionInstalled() {
+        try (var output = new ByteArrayOutputStream();
+                var exec = kubeClient().getClient().pods()
+                        .inNamespace(deploymentNamespace)
+                        .withName(VAULT_POD_NAME)
+                        .writingOutput(output)
+                        .exec("sh", "-c", VAULT_CMD + " version")) {
+            exec.exitCode().join();
+            // version returned as: Vault v1.15.2 (blah blah), build blah
+            return output.toString().split(" ")[1].replace("v", "");
```

Comment:
split on whitespace rather than a space and allow 1 or more.  It'll be slightly more robust against inconsequential changes.
```suggestion
            return output.toString().split("\\s+")[1].replace("v", "");
```